testsuite/tests/kuadrant/limitador/method/test_route_different_method.py

```diff
@@ -0,0 +1,41 @@
+"""Tests that RLP for a HTTPRouteRule doesn't affect the HTTPRoute with same path but different method"""
```

Comment:
I do not understand this package, I see no reason why have it in a separate package, there is no fixture that has package scope

Reply:
We discussed wanting more tests for specific methods with @jsmolar, we can discuss the need for it when he gets back and have this on hold meanwhile.